tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -1076,6 +1076,86 @@ StatusOr<bool> IsIdentityDrivingConstsInLoop(Node* node) {
   return true;
 }
 
+const absl::flat_hash_map<string, std::vector<string>> whitelist_table = {
```

Comment:
This looks like a non-trivially-destructible global, Google style guide prohibits those (cf. https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables)

Reply:
Fixed.